build/tasks.js

```diff
@@ -1,41 +1,46 @@
 // Any copyright is dedicated to the Public Domain.
 // http://creativecommons.org/publicdomain/zero/1.0/
 
-'use strict';
-
-module.exports = {
+/**
+ * Use need to allow lazy loading of modules for all tasks. Generally,
+ * we assume `build-config.js` exists, however *during* the build process
+ * this might not be true, which will break certain native module imports that
+ * try to eagerly load this file.
+ */
+
+export default {
   async config(options) {
-    await require('./task-config-builder')(options);
+    await require('./task-config-builder').default(options);
```

Comment:
Inline requires are still allowed, since the `import` eslint plugin assumes they're function calls. There's a few more occurrences of this in app/ code. Not sure what I could do about this, except disallow naming functions "require", or not assuming a node environment anymore.